cmd/juju/resource/upload.go

```diff
@@ -58,12 +58,34 @@ const (
 	attachDoc = `
 This command updates a resource for an application.
 
-For file resources, it uploads a file from your local disk to the juju controller to be
-streamed to the charm when "resource-get" is called by a hook.
+The format is
+
+    <resource name>=<resource>
```

Comment:
This duplications, maybe contradicts the usage message for attach-resource. Perhaps its better to change there?

Reply:
Fixed.